src/main/java/com/example/offsideoutfits/entity/User.java

```diff
@@ -0,0 +1,22 @@
+package com.example.offsideoutfits.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer userId;
```

Comment:
User ID could perhaps be a string? Could make it easier to identify as an ID if we changed the data type and prefixed it with a special character, maybe #

Reply:
Yeah I believe @GeneratedValue only works with Integer not with String

Reply:
I'm sure it's possible to make it work with a String - even if it's just prefixing the ID with a special character - but it may not be worth the added effort if it's not native to the @GeneratedValue tag. Looks good then!